.github/workflows/release.yaml

```diff
@@ -84,12 +84,24 @@ jobs:
         yq -i '(.releaseSeries[] | select(.contract == "v1beta1")).major |= env(MAJOR)' metadata.yaml
         yq -i '(.releaseSeries[] | select(.contract == "v1beta1")).minor |= env(MINOR)' metadata.yaml
 
+    - name: generate image info
```

Comment:
should we move this just above the list of artifacts instead of at the top.

Reply:
not possible easily, we need to entirely change the way we generate the release notes if we want to do that